src/model/document.ts

```diff
@@ -247,9 +247,8 @@ export class Document implements Feature {
   }
 
   /**
-   * Return all local features contained within the document as well as features
-   * detected by following through all imports. If the deep argument is set to
-   * false, only return local features.
+   * Get features for this document. If `deep` is true, also get features for
```

Comment:
This might lead people in the wrong direction, as `deep` defaults to true here. 

I wrote the suggestion for getWarnings thinking that we'd go with `deep` defaulting to false there, so maybe we should say something like:

    Get features reachable by this document and its imports. If `deep` is false, only get features contained in this document.